packages/app/healthcheck-utils/package.json

```diff
@@ -32,12 +32,14 @@
     "toad-cache": "^3.7.0"
   },
   "peerDependencies": {
+    "@lokalise/background-jobs-common": ">=5.0.0",
     "ioredis": "^5.4.1",
     "prom-client": "^15.1.3",
     "toad-scheduler": "^3.0.1"
   },
   "devDependencies": {
     "@biomejs/biome": "^1.8.3",
+    "@lokalise/background-jobs-common": "^5.2.0",
```

Comment:
As this is a `major` update, i think we should do this in a different PR to not trigger release

Reply:
bah, good catch, this leaked from a different set of changes that I was doing